<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewComDeviceDialog</class>
 <widget class="QDialog" name="NewComDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口设备</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasicSetting">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>帧间隔：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="editFrameTimePeriod">
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editFrameLen">
            <property name="text">
             <string>64</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>状态量：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="4">
           <widget class="QLineEdit" name="editProtocol">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QPushButton" name="btnProtocolSelect">
            <property name="text">
             <string>选择</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>帧长度：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>设备名称：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>通讯协议：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="5">
           <widget class="QLineEdit" name="editCtrlVar"/>
          </item>
          <item row="6" column="4">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>控制量：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cboLink">
            <item>
             <property name="text">
              <string>主链路</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>备用链路</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QPushButton" name="btnDeviceSelect">
            <property name="text">
             <string>选择</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="editDeviceName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>冗余配置：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QLineEdit" name="editStateVar"/>
          </item>
          <item row="5" column="3">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkDynamicOptimization">
           <property name="text">
            <string>使用动态优化</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>远程端口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editRemotePort"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPropertySetting">
      <attribute name="title">
       <string>设备属性</string>
      </attribute>
      <layout class="QVBoxLayout" name="vLayoutPropertyEditor"/>
     </widget>
     <widget class="QWidget" name="tabPortSetting">
      <attribute name="title">
       <string>端口配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="1">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>超时：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="cboPortNumber"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>串口号：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="editTimeout"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>数据位：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="cboStopbit"/>
          </item>
          <item row="4" column="2">
           <widget class="QComboBox" name="cboVerifybit"/>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="cboDatabit"/>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="cboBaudrate"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>停止位：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>校验位：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheck">
       <property name="text">
        <string>检查</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
